data-prepper-script/src/main/java/org/opensearch/dataprepper/script/parser/LogListener.java

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
I agree with the comments below.

Additionally, is there a reason this is in `main` and not `test`?